k

```diff
@@ -26,9 +26,24 @@ if [ ! -z "$KUBECONFIG" ]; then
   TARGET_VERSION=$(cat "$VERSION_CACHE_FILE" 2> /dev/null)
 fi
 
+# Ensure we have at least 1 kubectl version
+KNOWN_VERSION="v1.16.3"
+KNOWN_TARGET=$KX_PATH/kubectl-$KNOWN_VERSION
```

Comment:
nit: the var name `KNOWN_TARGET` is a bit abstract, something like `LOCAL_KUBECTL` or something that indicates "hey, this is the filepath of a known-working, version-agnostic kubectl binary that I'm going to use because I have to have *some* kubectl as a starting point to call the apiserver to determine if I, um, have the wrong  version and am actually not able to reliably communicate with the apiserver".

Chicken, meet egg.

lgtm otherwise

Reply:
Thanks!